docs/language/lateral-subqueries.md

```diff
@@ -7,11 +7,19 @@ sidebar_label: Lateral Subqueries
 
 Lateral subqueries provide a powerful means to apply a Zed query
 to each subsequence of values generated from an outer sequence of values.
-The inner query may be _any Zed query_ and may refer to values from
+The inner query may be _any_ dataflow operator sequence (excluding
+[`from` operators](operators/from.md)) and may refer to values from
 the outer sequence.
 
+:::tip Note
+This pattern rhymes with the SQL pattern of a "lateral
+join", which runs a SQL subquery for each row of the outer query's table.
```

Comment:
Nit: SQL seems redundant here. And "results" is clearer than "table".
```suggestion
join", which runs a subquery for each row of the outer query's results.
```

Reply:
@nwt: Works for me! That was actually existing text that I was just moving and reformatting. In general I am not too assertive about overhauling other people's statements about SQL, so I appreciate the suggestion.